src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Id.MachineName.cs

```diff
@@ -3,11 +3,19 @@
     public partial class Activity
     {
         #region private
-        private string generateUniquePrefix()
```

Comment:
`Environment.MachineName` is available from netstandard1.5, so `generateUniquePrefix` has different implementation on different versions. This is the reason to have them in two files and include proper file depending on the platform.